usage/test-support/src/main/java/org/apache/brooklyn/test/EntityTestUtils.java

```diff
@@ -46,20 +45,25 @@
 import com.google.common.collect.Maps;
 import com.google.common.collect.Sets;
 
+/**
+ * A utility class containing tests on Entities.
+ *
+ * @deprecated Prefer core assertions class org.apache.brooklyn.core.entity.EntityAsserts.
```

Comment:
Can you include a "since 0.9.0" in the deprecated javadoc line. Same for anywhere else you've added `@deprecated`.